pact/Pact/Core/Info.hs

```diff
@@ -22,3 +23,15 @@ instance Default SpanInfo where
 combineSpan :: SpanInfo -> SpanInfo -> SpanInfo
 combineSpan (SpanInfo l1 c1 _ _) (SpanInfo _ _ l2 c2) =
   SpanInfo l1 c1 l2 c2
+
```

Comment:
We have this as part of the LSP: https://github.com/kadena-io/pact-5/blob/b3f2c86afde946086fb123f100d2352230ff0e64/pact-lsp/Pact/Core/LanguageServer/Utils.hs#L104